canonical/juju-dashboard
# WD-9028 - Dedupe API hooks

## Done

- Combine common api hook handlers.

## QA

- Go to the secrets tab and check the secrets load.
- Check that you can perform write actions on the secrets.

## Details

- https://warthogs.atlassian.net/browse/WD-9028
